prow/cluster/jobs/istio/test-infra/istio.test-infra.trusted.master.yaml

```diff
@@ -95,11 +95,11 @@ periodics:
       - name: PLANK_DEPLOYMENT_FILE
         value: prow/cluster/plank_deployment.yaml
       - name: COMPONENT_FILE_DIR
-        value: prow/cluster,prow/config/jobs,.
+        value: prow/cluster,prow/cluster/jobs/istio/test-infra
```

Comment:
We need to bump this image with the rest of the prow components:
https://github.com/istio/test-infra/blob/89fa0e1fae5964c33780c59583b785a05c2e0d91/.prow.yaml#L8

Reply:
I need to either change ownership of the file or move this job back to `...jobs/istio/test-infra/` directory. I will do the latter.

Reply:
Updated code and modified CODEOWNERS.